rio_tiler/colormap.py

```diff
@@ -25,7 +25,7 @@
 EMPTY_COLORMAP: Dict = {i: [0, 0, 0, 0] for i in range(256)}
 
 DEFAULT_CMAPS_FILES = {
-    f.stem: str(f) for f in (resources_files(__package__) / "cmap_data").glob("*.npy")
+    f.stem: str(f) for f in (resources_files(__package__) / "cmap_data").glob("*.npy")  # type: ignore
```

Comment:
seen this also in timvt https://github.com/developmentseed/timvt/pull/58/commits/aab6c5b0d7d779e808009ac7aac10d0b4f60abbe
For now we can ignore but it would be good to revisit to see if there is a better way